modules/feed/src/main/FeedUi.scala

```diff
@@ -15,7 +15,7 @@ final class FeedUi(helpers: Helpers, atomUi: AtomUi)(
 
   private def renderCache[A](ttl: FiniteDuration)(toFrag: A => Frag): A => Frag =
     val cache = lila.memo.CacheApi.scaffeineNoScheduler
-      .expireAfterWrite(1 minute)
```

Comment:
We pass `tll` as `1 minute`, so remove the hard coded value

Reply:
but we still need `.expireAfterWrite(ttl)`